src/pages/catalog/elements/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import Head from "next/head";
+import Link from "next/link";
+import Image, { ImageProps } from "next/image";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import { RequestCourseModal } from "../../../components/RequestCourseModal";
+
+import styles from "./index.module.scss";
+import oerTruthTable from "../../../lib/images/oer-element-truthtable.png";
+import oerChemInput from "../../../lib/images/oer-element-cheminput.png";
+
+interface SharedElementCardProps {
+  image: ImageProps["src"];
+  title: string;
+  href: string;
+  ownerName?: string;
+  ownerEmail?: string;
+  github?: string;
+}
+
+const SharedElementCard: React.FC<SharedElementCardProps> = ({
+  image,
+  title,
+  href,
+  ownerName,
+  ownerEmail,
+  github,
+}) => {
+  return (
+    <article className="card border-secondary overflow-hidden">
+      <Link href={href} className="position-relative">
+        <Image
+          src={image}
+          alt={title}
+          style={{
+            objectFit: "contain",
+            width: "100%",
+            height: "100%",
+            aspectRatio: "5 / 3",
+          }}
+        />
+      </Link>
+      <div className="card-body">
+        <Link href={href}>
+          <h3 className="card-title h5">{title}</h3>
+        </Link>
+        {ownerName && (
+          <p className="mb-1">
+            <strong>Created by:</strong> {ownerName}{" "}
+            {ownerEmail && <a href={`mailto:${ownerEmail}`}>{ownerEmail}</a>}
+          </p>
+        )}
+        {github && (
+          <p className="mb-1">
+            <Link href={github}>Access on Github</Link>
+          </p>
+        )}
+      </div>
+    </article>
+  );
+};
+
+export default function Courses() {
+  const [showRequestCourseModal, setShowRequestCourseModal] =
+    React.useState(false);
+  return (
+    <React.Fragment>
+      <Head>
+        <title>OER Question Elements | PrairieLearn</title>
+      </Head>
+
+      <PageBanner
+        title="Open Educational Resources (OER): Question Elements"
+        subtitle="A catalog of building blocks that allow you to create richer questions"
+      />
+
+      <div className="container-fluid my-3">
+        <div className="container-md">
+          <div className="alert alert-secondary mb-0">
+            <p>
+              The shared question elements on this page are provided as-is and
+              not maintained by PrairieLearn. They are only recommended for
+              instructors who are already familiar with the platform. To use an
+              element, you will have to access its Github repository and
+              manually copy the element into your course.
+            </p>
```

Comment:
Nit, to avoid extra bottom margins:

```suggestion
            The shared question elements on this page are provided as-is and
            not maintained by PrairieLearn. They are only recommended for
            instructors who are already familiar with the platform. To use an
            element, you will have to access its GitHub repository and
            manually copy the element into your course.
```